### ISSUE TITLE
Feature Request: Virtual Background & Frame Processing with AI Models

### ISSUE THREAD

**Issue by boussaidev** [react-native-webrtc/react-native-webrtc#1642]
.

**Comment by boussaidev** [comment]
this is an example (https://getstream.io/video/docs/reactnative/advanced/apply-video-filters/)[https://getstream.io/video/docs/reactnative/advanced/apply-video-filters/]

**Comment by saghul** [comment]
They use a fork of this plugin lol.

With an iOS patch which we have in a branch.

Lame GetStream, lame.

**Comment by boussaidev** [comment]
@saghul  is it possible to implement this feature ? :-D

**Comment by saghul** [comment]
Yes, but not today in master, since we need to land the iOS part.

**Comment by boussaidev** [comment]
@saghul anything new ? i can mention a repo so we can have an idea about the code !
i really want to participate on this but i have no idea about native code

**Comment by boussaidev** [comment]
@saghul i think having an idea from this repo will make things really quick

**Comment by saghul** [comment]
Sorry, it looks like it's not at a stage in which you can help. Pay attention to the release notes, it'll be announced when it's ready.

**Comment by boussaidev** [comment]
@saghul i think this can be helpfull https://github.com/GetStream/stream-video-js/tree/main/packages/react-native-sdk https://github.com/GetStream/stream-video-js/tree/main/packages/video-filters-react-native

**Comment by boussaidev** [comment]
@saghul please is there is any update ?
